packages/docusaurus/src/client/docusaurus.js

```diff
@@ -18,7 +18,7 @@ const isSlowConnection = () => {
   // if user is on slow or constrained connection
   if (`connection` in navigator) {
     if (
-      (navigator.connection.effectiveType || ``).includes(`2g`) &&
+      (navigator.connection.effectiveType || ``).indexOf(`2g`) !== -1 &&
```

Comment:
The only difference is that includes(NaN) and indexOf(Nan) !== -1 is different. But `2g` is hardcoded anyway so there's no diff in this function